tools/ccc

```diff
@@ -64,6 +67,9 @@ if [ $# -eq 0 ]; then
 	usage
 fi
 
+arch=$1
+shift
```

Comment:
I think we need to move the setting of SYSROOT below here as it's currently:
```
SYSROOT=${SYSROOT:-${CHERIBUILD_SDK}/sysroot-riscv64-purecap}
```

Reply:
Oops, yes, I was in a FETT mindset where it's just called sysroot. I guess that should be `sysroot-${arch%%-*}-purecap` too?

Reply:
I don't have a build around, but that's what I'd expect. There might also be some issues with finding the right clang if we end up having two llvm's around.

Reply:
Yeah... emphasis on "preliminary" :)

Reply:
The sysroots for Morello produced by cheribuild are `${CHERIBUILD_SDK}/sysroot-aarch64`, `${CHERIBUILD_SDK}/sysroot-morello-hybrid`, and `${CHERIBUILD_SDK}/sysroot-morello-purecap`.

Could the default value for `${SYSROOT}` be `${CHERIBUILD_SDK}/sysroot-${arch}` when `${arch}` is any of the riscv64 flavors or aarch64, and `${arch/aarch64/morello}` for morello hybrid and purecap?

Reply:
That's an annoying decision on cheribuild's part. For the sake of consistency I think they should be changed to all say aarch64.